torch/csrc/jit/tensorexpr/expr.h

```diff
@@ -313,6 +364,13 @@ class TORCH_API BufHandle : public ExprHandle {
       Dtype dtype)
       : ExprHandle(Buf::make(name_hint, dims, dtype)) {}
 
+  BufHandle(
+      const std::string& name_hint,
+      const std::vector<ExprHandle>& dims,
+      const std::vector<ExprHandle>& strides,
+      Dtype dtype)
+      : ExprHandle(Buf::make(name_hint, dims, strides, dtype)) {}
+
```

Comment:
Let's also expose `is_contiguous` and `is_channels_last_1d_contiguous` here in `BufHandle` to keep its API in sync with API of `Buf`. In the end, it's just a simple wrapper.